src/App.vue

```diff
@@ -950,10 +950,6 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // set filing type and status
     this.setFilingType(draftFiling.header.name)
     this.setFilingStatus(draftFiling.header.status)
-    // *** TODO: remove after testing
-    if (draftFiling.header.status === FilingStatus.CHANGE_REQUESTED) {
-      this.setFilingStatus(FilingStatus.APPROVED)
-    }
```

Comment:
This was a simple work-around so that "Change Requested" continuation filings would appear as Approved and therefore show steps 1-4 (instead of just step 0 aka authorization step).

Note that it's not possible to display Approved filings in the Todo List, because the BE currently sends them to the Filer. The new BE code is expected to me merged today. At that point, we should be able to see Approved continuation filing in the Todo List, and resume them in Create UI to complete steps 1-4, as per the new flow.